### PR TITLE
Fix vsce rewriting and refactor tests

### PR DESCRIPTION
**Pull request by Discookie**
The rewriter for publishing to the Marketplace also needs to rewrite a couple things within `yarn.lock`.

Updated the GH Actions files to use the new output syntax, and refactored tests - they now also test `vsce package` on the plugin.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -1,24 +1,29 @@
-name: codechecker-vscodeplugin-tests
+name: Run plugin tests
 
 # Triggers the workflow on push or pull request events.
```
Comment (vodorok) [inline .github/workflows/test.yml:3]: Is this comment still apply?

File: .github/workflows/package.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+name: Package CodeChecker VSCode extension
+
+# Triggers the workflow on push or pull request events.
```
Comment (vodorok) [inline .github/workflows/package.yml:3]: Triggers on workflow calls also, instead of push or pull request.

**Review by vodorok — CHANGES_REQUESTED** [review]
Please see my remarks

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -6,12 +6,26 @@ on:
     types: [published]
 
 jobs:
+  version:
+    name: Version dummy
```
[the remark this answers is not in the document]
Comment (Discookie) [reply .github/workflows/deploy.yml]: Indeed it is not, fixed.

File: .github/workflows/test.yml (hunk 1 above)
Comment (Discookie) [reply .github/workflows/test.yml]: Whoops, slight ctrl-C issue, fixed.

**Review by vodorok — APPROVED** [review]
LGTM

**Pull request merged**